src/pathfinding_service/model/channel_view.py

```diff
@@ -19,22 +23,25 @@ class State(Enum):
         SETTLING = 2
         SETTLED = 3
 
+    token_network_address: TokenNetworkAddress
     channel_id: ChannelID
     participant1: Address
     participant2: Address
     settle_timeout: int
-    capacity: int = field(init=False)
+    capacity: int = None  # type: ignore
     reveal_timeout: int = DEFAULT_REVEAL_TIMEOUT
     deposit: TokenAmount = TokenAmount(0)
-    state: State = State.OPEN
-    update_nonce: int = field(default=0, init=False)
-    absolute_fee = FeeAmount(0)
+    state: State = State.OPEN  # TODO: do we care about channels with state != OPEN?
```

Comment:
Good point, is this used/checked anywhere?

Reply:
No, and it's removed in another commit of this PR.